src/App.css

```diff
@@ -1,23 +1,36 @@
+* {
+  background-color: antiquewhite;
+}
 h1 {
   text-align: center;
   font-family: 'Chalkduster';
+  font-size: 35pt;
 }
 
 h2 {
   text-align: center;
   font-family: 'Impact';
 }
 
-h3 {
-  margin: auto;
-  line-height: 200px;
+.room {
+  display: flex;
+  justify-content: space-evenly;
   text-align: center;
-  vertical-align: middle;
+}
+
+.lightswitch {
+  margin-left: auto;
+  margin-right: auto;
+  height: 200px;
+  width: 200px;
+  background-size: contain;
 }
 
 .lightbulb {
-  border: 2px solid black;
+  margin-left: auto;
+  margin-right: auto;
   height: 200px;
   width: 200px;
-  margin: 1rem auto; 
+  background-size: contain;
```

Comment:
These two classes are almost identical. You could make one class that contains all the common properties and then another one that only contains the differences and then chain them together to get the right combination. Here is an article that talks about [chaining](https://frontstuff.io/should-you-chain-or-extend-css-classes).

Reply:
I appreciate your feedback.  I will follow up, and see if I can refactor.